### PR TITLE
Issue #144 add start and end time to site

### PR DESCRIPTION
**Pull request by micah-prime**
Issue #144 
Add a start and end time for pit start and end to Site

### CODE REVIEW

File: snowexsql/tables/site.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ class Site(SingleLocationData, Base, InCampaign, HasDOI):
     vegetation_height = Column(String())
     tree_canopy = Column(String())
     site_notes = Column(String())
+    start_time = Column(Time)
+    end_time = Column(Time)
```
Comment (jomey) [inline snowexsql/tables/site.py:57]: Should this be timezone aware?

**Review by jomey — CHANGES_REQUESTED** [review]
With the new testing infrastructure, I think this change should also update the factory `tests/factory` and table level tests under `tests/tables/`

**Comment by micah-prime** [comment]
> With the new testing infrastructure, I think this change should also update the factory `tests/factory` and table level tests under `tests/tables/`

good call

File: snowexsql/tables/site.py (hunk 1 above)
Comment (micah-prime) [reply snowexsql/tables/site.py:57]: Oh yes, right!

**Pull request merged**